camayoc/tests/qpc/cli/test_sources.py

```diff
@@ -72,8 +72,10 @@
 
 
 def generate_show_output(data):
-    """Generate a regex pattern with the data for a qpc cred show output."""
+    """Generate a regex pattern that matches `qpc source show` output (API v2)."""
     output = r"{\r\n"
+
+    # ─ credentials ─
```

Comment:
How strongly do you feel about keeping or dropping these extra inline comment explanations? I understand if this could improve readability in complex situations, but it looks like all of these commented words are almost immediately followed by the same word in the next line of code (e.g. `# ─ disable_ssl ─` followed by `if "disable_ssl" in data:`) which makes the comments largely redundant.

Reply:
I'm not attached to them at all. I added those inline comments just to help organize the code initially, since the whitespace and line breaks in this function were pretty messy. But now that everything's working, we can definitely drop them!

Reply:
Our CLI handling in general is a mess, and that function is one of the worst offenders. Whoever though checking JSON output with hand-crafted regex is a good idea? We should have refactored that by now, but you know, if it ain't broke...